### ISSUE TITLE
error minify inline svg

### ISSUE THREAD

**Issue by liu-dongyu** [jakubpawlowicz/clean-css#776]
When trying minify inline svg css file, it get error. 

My svg is valid in chrome.

input 

```css
background-image: url('data:image/svg+xml;charset=utf-8,<svg viewBox=\'0 0 120 120\' xmlns=\'http://www.w3.org/2000/svg\' xmlns:xlink=\'http://www.w3.org/1999/xlink\'><defs><line id=\'l\' x1=\'60\' x2=\'60\' y1=\'7\' y2=\'27\' stroke=\'%236c6c6c\' stroke-width=\'11\' stroke-linecap=\'round\'/></defs><g><use xlink:href=\'%23l\' opacity=\'.27\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(30 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(60 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(90 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(120 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(150 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.37\' transform=\'rotate(180 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.46\' transform=\'rotate(210 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.56\' transform=\'rotate(240 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.66\' transform=\'rotate(270 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.75\' transform=\'rotate(300 60,60)\'/><use xlink:href=\'%23l\' opacity=\'.85\' transform=\'rotate(330 60,60)\'/></g></svg>');
```

output

```css
background-image:url("data:image/svg+xml;charset=utf-8,<svg viewBox=\'0 0 120 120\' xmlns=\'http://www.w3.org/2000/svg\' xmlns:xlink=\'http://www.w3.org/1999/xlink\'><defs><line id=\'l\' x1=\'60\' x2=\'60\' y1=\'7\' y2=\'27\' stroke=\'%236c6c6c\' stroke-width=\'11\' stroke-linecap=\'round\'/></defs><g><use xlink:href=\'%23l\' opacity=\'.27\'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(30 60,60") \'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(60 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(90 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(120 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.27\' transform=\'rotate(150 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.37\' transform=\'rotate(180 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.46\' transform=\'rotate(210 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.56\' transform=\'rotate(240 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.66\' transform=\'rotate(270 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.75\' transform=\'rotate(300 60,60) \'/><use xlink:href=\'%23l\' opacity=\'.85\' transform=\'rotate(330 60,60) \'/></g></svg>');
```

**Comment by jakubpawlowicz** [comment]
Thanks @liu-dongyu, it's a bug indeed.

**Comment by jakubpawlowicz** [comment]
Fixed in 3.4.15.

**Comment by elfin-sbreuers** [comment]
After rolling out version 3.4.15 of clean-css I get the following error:

Input file test.scss:

```css
.crossed {
    background-image: url('data:image/svg+xml;utf8,<svg xmlns="http://www.w3.org/2000/svg" version="1.1" preserveAspectRatio="none" viewBox="0 0 100 100"> <line x1="0" y1="0" x2="100" y2="100" style="stroke:rgb(50, 50, 50);stroke-width:1" /><line x1="0" y1="100" x2="100" y2="0" style="stroke:rgb(50, 50, 50);stroke-width:1" /></svg>');
}
```

Application call:

```bash
cleancss -v && cleancss test.scss
```

Output:

```bash

3.4.15
WARNING: Missing '}' after '
    background-image: __ESCAPED_URL_CLEAN_CSS0__;stroke-width:1__ESCAPED_FREE_TEXT_CLEAN_CSS0__0__ESCAPED_FREE_TEXT_CLEAN_CSS1__100__ESCAPED_FREE_TEXT_CLEAN_CSS2__100__ESCAPED_FREE_TEXT_CLEAN_CSS3__0__ESCAPED_FREE_TEXT_CLEAN_CSS4__stroke:rgb(50, 50, 50);stroke-width:1__ESCAPED_FREE_TEXT_CLEAN_CSS5__'. Ignoring.

```

Do you have an idea of how to solve it in CSS or is this a bug in clean-css?

Kind regards


Sebastain
